python/graphframes/tests.py

```diff
@@ -227,6 +261,18 @@ def test_page_rank(self):
         pr = g.pageRank(resetProb, tol=errorTol)
         self._hasCols(pr, vcols=['id', 'pagerank'], ecols=['src', 'dst', 'weight'])
 
+    def test_parallel_personalized_page_rank(self):
+        if GraphFrameTestUtils.is_later_version("2.1"):
```

Comment:
It is a bit tricky as determining current spark version requires the `sc` object. When using if with `@unittest.skipIf` annotation, an error is reported. It looks like `sc` is not initialized when the annotation is evaluated.